NEXT_CHANGELOG.md

```diff
@@ -32,4 +32,42 @@ By [@USERNAME](https://github.com/USERNAME) in https://github.com/apollographql/
 
 For `experimental_cache` with redis caching it now works with only a single Redis instance if you provide only one URL.
 
-By [@bnjjj](https://github.com/bnjjj) in https://github.com/apollographql/router/pull/2310
\ No newline at end of file
+By [@bnjjj](https://github.com/bnjjj) in https://github.com/apollographql/router/pull/2310
+
+### Override the root certificate list for subgraph requests ([Issue #1503](https://github.com/apollographql/router/issues/1503))
+
+we might want to connect over TLS to a subgraph with a self signed certificate, or using a custom certificate authority.
+This adds a configuration option to set the list of certificate authorities for all the subgraphs, as follows:
+
+```yaml
+tls:
+  subgraphs:
+    certificate_authorities_path: /path/to/ca.crt
```

Comment:
Can we consider not having this as a path and instead using file expansion via the config system to inject? That way if we want to disable file system access for the router in secure environments we can.

```yaml
tls:
  subgraphs:
    certificate_authorities_path: "${file./path/to/ca.crt}"
```
See https://www.apollographql.com/docs/router/configuration/overview/#variable-expansion

Reply:
oh that looks nice!

Reply:
done in d2c9b5da